test/spec/features/ContextPadTracking.spec.js

```diff
@@ -75,6 +77,34 @@ describe('ContextPadTracking', function() {
       });
     }));
 
+
+    it('get closest "data-action"', inject(function(elementRegistry, selection, bpmnJSTracking) {
+
+      // given
+      const element = elementRegistry.get('StartEvent_1');
+      selection.select(element);
+
+      const spy = sinon.spy();
```

Comment:
Usually a cleaner way to do this is to make the spy an actual function, and asser the correct arguments in there:

```javascript
const spy = sinon.spy(function(event) {
  expect(event).to.eql({...});
});


// ...later in the tests
expect(spy).to.be.calledOnce;
```

Reply:
I added a commit to clean that up

Reply:
This merged automatically. There are some more instances of this. Since this is only test-related code style, I'm going to go ahead and release to not block the release and have a look at cleaning this up completely tomorrow.